dotnetv3/Organizations/README.md

```diff
@@ -0,0 +1,55 @@
+# AWS Organizations code examples for .NET
+
+## Overview
+
+The examples in this section show how to use the AWS SDK for .NET with AWS Organizations to create and manage AWS Organizations policies.
+
+AWS Organizations helps you centrally manage and govern your environment as you grow and scale your AWS resources. Using AWS Organizations, you can programmatically create new AWS accounts and allocate resources, group accounts to organize your workflows, apply policies to accounts or groups for governance, and simplify billing by using a single payment method for all of your accounts.
```

Comment:
With Organizations, you can consolidate multiple AWS accounts into an organization that you create and centrally manage. You can create member accounts and invite existing accounts to join your organization. You can organize those accounts and manage them as a group.

Reply:
Fixed.